MAUI/DataGrid/selection.md

```diff
@@ -520,4 +543,19 @@ public class ViewModel : INotifyPropertyChanged
         }
     }
 {% endhighlight %}
+{% endtabs %}
+
+## Set KeyBoard Navigation
+
+SfDataGrid allows you to set the Keyboard Navigation property as `True` or `False`. By default, Keyboard Navigation works. But you can set it to `False`, if you prefer to deactivate it.
```

Comment:
By default, the keyboard navigation will be enabled when setting the selection and navigation modes. It can be disabled by setting the `SfDataGrid.AllowKeyboardNavigation` to `false`.

Reply:
Changed